<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendStuffToShop</class>
 <widget class="QDialog" name="SendStuffToShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ارسال کالا به فروشگاه</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>نام کالا:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>به فروشگاه مرکزی</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>از انبار:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>حداقل موجودی:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>بیخیال</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>ارسال</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>تعداد:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SendStuffToShop</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
